internal/command/meta.go

```diff
@@ -736,6 +736,27 @@ func (m *Meta) showDiagnostics(vals ...interface{}) {
 	}
 }
 
+const (
+	// StatePersistIntervalEnvVar is the environment variable that can be set
+	// to control the interval at which Terraform persists state. The interval
+	// itself defaults to 20 seconds.
+	StatePersistIntervalEnvVar = "TF_STATE_PERSIST_INTERVAL"
+)
+
+// StatePersistInterval returns the configured interval that Terraform should
+// persist statefiles to the desired backend. Backends may choose to override
+// the default value.
+func (m *Meta) StatePersistInterval() int {
+	if val, ok := os.LookupEnv(StatePersistIntervalEnvVar); ok {
+		interval, err := strconv.Atoi(val)
+		if err == nil {
+			return interval
+		}
+		log.Printf("[ERROR] Can't parse persist interval '%s' of environment variable '%s'", val, StatePersistIntervalEnvVar)
```

Comment:
💅 standard quote formatting

```suggestion
		log.Printf("[ERROR] Can't parse persist interval %q of environment variable %q", val, StatePersistIntervalEnvVar)
```